projects/storefrontlib/src/shared/utils/validators/custom-form-validators.ts

```diff
@@ -140,6 +146,53 @@ export class CustomFormValidators {
 
     return !containsSpecialChars ? null : { cxContainsSpecialCharacters: true };
   }
+
+  /**
+   * TODO
+   */
+  static datePatternValidation(
+    service: DatePickerService,
```

Comment:
I won't like to put in a complete service in an argument. What about pass in `isValidFormat: boolean` instead?